app/services/client.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from dotenv import load_dotenv
+from os import getenv
+
+
+load_dotenv()
```

Comment:
you can use pydantic base_settings to have more control over the env variables.

https://docs.pydantic.dev/latest/concepts/pydantic_settings/
you can take a look here at the documentation,

you can create a settings.py file in the app directory and provide the env variables you want to define in the BaseSettings class exported from pydantic_settings

Reply:
I didn't understand anything from there